framework/src/controller/bus.ts

```diff
@@ -250,6 +248,7 @@ export class Bus extends EventEmitter2 {
 		if (this.config.ipc.enabled) {
 			this.subSocket?.on(eventName, cb);
 		}
+		return this;
```

Comment:
Do we want function chainning on this function?

Reply:
this was required because it's extending the EventEmitter2, and signature has to be exactly the same